include/bluetooth/services/hids_c.h

```diff
@@ -243,7 +206,25 @@ struct bt_gatt_hids_c_init_params {
 };
 
 /**
- * HIDS client structure
+ * @brief Report base information struct
+ *
+ * The structure that contains record information.
+ * This is common structure for Input and Output/Feature reports.
+ * Some of the fields are used only for Input report type.
+ *
+ */
+struct bt_gatt_hids_c_rep_info;
+
+/**
+ * @brief HIDS client structure
```

Comment:
The doc build throws errors here:

```
ervices\hids_c.rst:115: WARNING: Invalid definition: Expected identifier in nest
ed name. [error at 23]
  struct bt_gatt_hids_c::@0  bt_gatt_hids_c::handlers
  -----------------------^
C:\Users\rufu\Desktop\GIT\ncs\nrf\doc\_build\nrf\rst\doc\nrf\include\bluetooth\s
ervices\hids_c.rst:115: WARNING: Invalid definition: Expected identifier in nest
ed name. [error at 23]
  struct bt_gatt_hids_c::@1  bt_gatt_hids_c::init_repref
  -----------------------^
C:\Users\rufu\Desktop\GIT\ncs\nrf\doc\_build\nrf\rst\doc\nrf\include\bluetooth\s
ervices\hids_c.rst:115: WARNING: Invalid definition: Expected identifier in nest
ed name. [error at 23]
  struct bt_gatt_hids_c::@2  bt_gatt_hids_c::rep_boot
  -----------------------^
C:\Users\rufu\Desktop\GIT\ncs\nrf\doc\_build\nrf\rst\doc\nrf\include\bluetooth\s
ervices\hids_c.rst:115: WARNING: Duplicate declaration.
```
I haven't started debugging to track down the reason, but maybe you have an idea about how this struct is different from others?

Reply:
It seems that there is an issue when an anonymous structure (handlers) is defined within the other structure (struct bt_gatt_hids_c)

Reply:
Managed to "fix" this by only documenting the members of the anonymous structure, not the structure itself.
The last issue that there are two members called "pm" - one of them must be renamed.